src/libsyntax/diagnostics/macros.rs

```diff
@@ -1,7 +1,7 @@
 #[macro_export]
```

Comment:
The remainder of this file should probably be moved to `librustc_errors`?

Reply:
They seem to be "independent"? A bit weird...